### ISSUE TITLE
Extra page size option: Exactly one divider

### ISSUE THREAD

**Issue by Womabre** [sumpfork/dominiontabs#330]
Some print services kan cut custom size paper, but they need a PDF where each page is exactly one divider.

**Comment by nickv2002** [comment]
@Womabre does the large-divider-gap solution proposed by @mrgrain work for your desired use case?

**Comment by Womabre** [comment]
@nickv2002 Yes, this works. If I crop the PDF afterwards I end up with a suitable file.
